proposal/aiiuii/README.rst

```diff
@@ -0,0 +1,93 @@
+==============================
+Interbotix Ros Xsarms Toolkits
+==============================
+
+
+
+Basic Information
+=================
+
+The GitHub repository: https://github.com/aiiuii/Interbotix_Ros_Xsarms_Toolkits
+
+Problem to Solve
+================
+
+Recently, my supervising professor purchased a robotic arm for the lab. Unfortunately, no one in the lab had any experience with it, so learning how to use it ended up being one of my professor's assigned tasks.
+
+For this project, I intended to create a GUI system that would allow users to control the robot arm. Although the vendor already had an example, it was too simple in its functionality.
+
+In this project I will use the following tools:
+
+* PyQt: 
+      To create GUI interface.
+* ROS: 
+      The Robot Operating System (ROS) is a set of software libraries and tools that help you build robot applications.
+* IRROS: 
+      The SDK developed by the vendor.It provides API to interact with robotic arm.
+* MoveIt: 
+      MoveIt is ROS's most advanced and flexible library for motion planning and manipulation tasks. It integrates state-of-the-art inverse kinematics solvers, path planning algorithms, and collision detection into a single, unified ROS interface.
+
+
+
+Prospective Users
+=================
+
+Anyone who needs to use robotic arm.
+
+System Architecture
+===================
+
+1. 
+The system will monitor ROS nodes created by IRROS, analyze data and show related information on the screen.
+
+2.
+The system will provides GUI to adjust target position and orientation.Then, users can perform moveit_planning command and robotic arm will follow it.
```

Comment:
GUI is also another challenge in engineering.  In practice, long work hours are a must for a good GUI implementation, and there are few smart ways to improve.